rpm/build/SPECS/jenkins.spec

```diff
@@ -8,14 +8,10 @@ Version:	%{ver}
 Release:	1.1
 Summary:	@@SUMMARY@@
 Source:		jenkins.war
-Source1:	jenkins.init.in
-Source2:	jenkins.sysconfig.in
 Source3:	jenkins.logrotate
 Source4:	jenkins.service
 Source5:	jenkins.sh
```

Comment:
Nice, but can we now please renumber these sources starting from the first number to reflect the fact that there are now only four sources.

Reply:
Sure, will do that.